test/test_torch.py

```diff
@@ -5932,7 +5932,7 @@ def test_grad_scaling_autocast_foreach(self, device):
         for optimizer_ctor in (torch.optim.SGD, torch.optim.Adam, torch.optim.AdamW):
             self._grad_scaling_autocast_test(device=device.type, optimizer_ctor=optimizer_ctor, optimizer_kwargs={"foreach": True})
 
-    @onlyCUDA
+    @onlyNativeDeviceTypes
     def test_grad_scaling_autocast_fused(self, device):
         device = torch.device(device)
         for optimizer_ctor in (torch.optim.Adam, torch.optim.AdamW):
```

Comment:
oh lol you're making me realize we totally forgot to add SGD to this list when fusedSGD got added (cc @crcrpar)--I'll add an issue to track this. https://github.com/pytorch/pytorch/issues/123451